README.md

```diff
@@ -119,6 +119,51 @@ For example, here are some acceptable values:
     GITHUB_USER     => elixir-lang
     GITHUB_REPO     => ecto
 
+## Using ExDoc with Erlang projects
+
+To use ExDoc with Erlang projects you need to do the following:
+
+1. Use OTP 24+
+
+2. Add the following to your `rebar.config`. This instructs `edoc` to generate doc chunks
+   instead of HTML docs:
+
+   ```erlang
+   {edoc_opts, [
+     {doclet, edoc_doclet_chunks},
+     {layout, edoc_layout_chunks},
+     {dir, "_build/default/lib/<app>/doc"}]}.
+   ```
+
+   Replace `<app>` with the name of your app.
+
+3. Install ExDoc escript:
+
+   ```bash
+   $ mix escript.install github elixir-lang/ex_doc
+   ```
+
+4. Generate docs:
```

Comment:
We need to say that, if the escript path is not in their system paths, they will need to point to it directly.